webpack/ForemanInventoryUpload/Components/ExcludePackagesSwitcher/ExcludePackagesSwitcher.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Switch, FieldLevelHelp } from 'patternfly-react';
+import { translate as __ } from 'foremanReact/common/I18n';
+import './excludePackagesSwitcher.scss';
+
+const ExcludePackagesSwitcher = ({ excludePackages, handleToggle }) => (
```

Comment:
I think it makes sense to refactor all switchers to a common component and reuse the same component with different params in `InventorySettings`.

Reply:
Sounds good, how about getting this feature in first
and than do in a follow-up the refactor for the 3 buttons?